# Generated by Django 5.0.6 on 2024-08-20 21:51

import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0001_initial'),
    ]

    operations = [
        migrations.AddField(
            model_name='user',
            name='phone',
            field=models.CharField(blank=True, default='', error_messages='Телефон нөмірі қате енгізілді.', max_length=15, null=True, validators=[django.core.validators.RegexValidator('\\+[0-9]{1}\\([0-9]{3}\\)[0-9]{3}\\-[0-9]{4}')]),
        ),
        migrations.AddField(
            model_name='user',
            name='subscribe',
            field=models.DateField(blank=True, default='2001-09-11', null=True),
        ),
    ]
